changelog/Microsoft.DirectoryServices.json

```
@@ -186,6 +186,40 @@
       "WorkloadArea": "Identity and access",
       "SubArea": "Identity and sign-in"
     },
+    {
+      "ChangeList": [
+        {
+          "Id": "dbd62dae-abc1-419d-8322-045d266f0b45",
+          "ApiChange": "Enumeration",
+          "ChangedApiName": "weakAlgorithms",
+          "ChangeType": "Addition",
+          "Description": "Added the [weakAlgorithms](https://docs.microsoft.com/en-us/?view=graph-rest-beta) resource",
```

Fixing a stub generator issue
```suggestion
          "Description": "Added the [weakAlgorithms](https://docs.microsoft.com/en-us/?view=graph-rest-beta) enumeration type.",
```